x-pack/metricbeat/module/aws/ebs/_meta/data.json

```diff
@@ -2,42 +2,43 @@
     "@timestamp": "2017-10-12T08:05:34.853Z",
     "aws": {
         "cloudwatch": {
-            "dimensions": {
-                "VolumeId": "vol-6ae467c0"
-            },
             "namespace": "AWS/EBS"
         },
-        "metrics": {
-            "BurstBalance": {
-                "avg": 100
-            },
-            "VolumeIdleTime": {
-                "sum": 299.97
-            },
-            "VolumeQueueLength": {
-                "avg": 0.0001
-            },
-            "VolumeReadOps": {
-                "avg": 0
-            },
-            "VolumeTotalWriteTime": {
-                "sum": 0.03
-            },
-            "VolumeWriteBytes": {
-                "avg": 10330.79802955665
-            },
-            "VolumeWriteOps": {
-                "avg": 203
+        "dimensions": {
+            "VolumeId": "vol-092bd3981e4e05170"
+        },
+        "ebs": {
+            "metrics": {
```

Comment:
I wonder if we still need this `metrics` part in the data now?

Reply:
Yeah that's a good question. I use it to avoid clashing with existing metricsets (`ec2`, `s3`, and others).

Reply:
Hmm what kind of clashing are we considering here? Like same metric names betwen elb and ec2 for example?

Reply:
nope, ec2 has it's own mapping, as it was developed before lightweight modules, so having a dynamic mapping for aws.*.* would clash with `aws.ec2.xxx` mappings, also s3 and others

Reply:
that said, it would be interesting to see if we can workaround that, dynamic templates are processed in order, so if this is the last one things should probably work :thinking:

Reply:
Ahh thanks for explaining. For example `aws.ec2.instance.state.name` won't fit for dynamic mapping because it's keyword defined in ec2 metricset not float.

Reply:
@exekias How about `aws.<namespace>.metrics.*` for using cloudwatch metricset as input? This will make the distinction between using cloudwatch metricset as input and other metricsets in aws module. This way, using ec2 as example, if we use light module, then the metrics will be `aws.ec2.metrics.CPUUtilization`. The same metric created from ec2 metricset will be `aws.ec2.cpu.total.pct`.

Reply:
that sounds good to me, where would you take that `namespace` from? Should the developer input it as a config parameter?

Reply:
@exekias I was thinking it would be the actual metricset name. For elb, the metric would be `aws.elb.metrics.EstimatedProcessedBytes`.

Reply:
I have been checking https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/aws-services-cloudwatch-metrics.html and realized: Wy not using the cloudwtch Namespace itself? Just remove `AWS/` and lowercase.